shuffle/cli/src/shared.rs

```diff
@@ -308,8 +315,18 @@ impl Home {
 
 pub fn normalized_network(home: &Home, network: Option<String>) -> Result<Url> {
     match network {
-        Some(input) => Ok(home.read_networks_toml()?.url_for(input.as_str())?),
-        None => Ok(home.read_networks_toml()?.url_for(LOCALHOST_NETWORK_NAME)?),
+        Some(input) => Ok(home
+            .read_networks_toml()?
+            .networks
+            .get(input.as_str())
```

Comment:
In another PR, are we creating a `get` that you can use directly on all_networks?

Reply:
added in this PR